pyRevitPlus.tab/Smart Align.panel/smartalign.stack3/Vertical.pulldown/Align Center.pushbutton/script.py

```diff
@@ -23,11 +23,11 @@
 
 if not VERBOSE:
   #__window__.Close()
-
+    
 # ALIGN = Align.HCENTER
 # ALIGN = Align.HLEFT
 # ALIGN = Align.HRIGHT
-ALIGN = Align.VCENTER
+    ALIGN = Align.VCENTER

```

Comment:
Why is this indented?

Reply:
Remove the indentation at line 30 and reload Pyrevit - the Reload window has an Error for unexpected indented block.

ERROR: [pyrevit.extensions.genericcomps] Error parsing script file: C:/Users/____pyRevitPlus.tab/Smart Align.panel/smartalign.stack3/Vertical.pulldown/Align Center.pushbutton/script.py | Error parsing script file: C:/Users/_____/pyRevitPlus.tab/Smart Align.panel/smartalign.stack3/Vertical.pulldown/Align Center.pushbutton/script.py | expected an indented block 

Traceback: 
 File "C:/Users/_____pyRevit/pyrevitlib/pyrevit/extensions/genericcomps.py", line 322, in _analyse_python_script 
 script_content = ScriptFileParser(self.get_full_script_address())